lessons/3_first_interactive_component/index.html

```diff
@@ -364,29 +364,92 @@ <h3>For that you need to do:</h3>
 					</li>
 				</ul>
 			</section>
+			<section>
+				<h3>But where do we write JavaScipt code?</h3>
+			</section>
+			<section>
+				<p>We will use <b>&lt;script/&gt;</b> tag inside <b>index.html</b></p>
+				<pre>
+					<code>
+&lt;html&gt;
+ &lt;head&gt;
+  &lt;meta charset="utf-8"&gt;
+	&lt;title&gt;My HTML Page&lt;/title&gt;
+	&lt;script type="text/javascript"&gt;
```

Comment:
I think the script should be in the **body** tag like you are using in the example.

Reply:
Done!